### PR TITLE
Dtspo 5327

### PR DESCRIPTION
**Pull request by darren221980**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-5327

### Change description ###

used to run azure-platform-terraform locally for creation of Demo AG's.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: environments/demo/demo.tfvars [hunk 1]
```diff
@@ -1,7 +1,8 @@
 env                       = "demo"
 subscription              = "demo"
-app_gw_private_ip_address = ["10.51.32.121", "10.51.32.125"]
-cft_apps_cluster_ips      = ["10.51.31.250"]
+app_gw_private_ip_address = ["10.50.98.121", "10.50.98.125"]
+cft_apps_cluster_ips      = ["10.50.64.250"]
+cft_apps_ag_ip_address    = "10.50.98.123"
```
[the remark this answers is not in the document]
Comment (darren221980) [reply environments/demo/demo.tfvars]: Thats for the front end agw used by this module - https://github.com/hmcts/azure-platform-terraform/blob/8c77a36bd4276cad7c072aa2b64122dbdb362732/components/cftapps_cluster_lb/main.tf#L16

File: environments/demo/demo.tfvars [hunk 2]
```diff
@@ -1,7 +1,8 @@
 env                       = "demo"
 subscription              = "demo"
-app_gw_private_ip_address = ["10.51.32.121", "10.51.32.125"]
-cft_apps_cluster_ips      = ["10.51.31.250"]
+app_gw_private_ip_address = ["10.50.98.121", "10.50.98.125"]
+cft_apps_cluster_ips      = ["10.50.64.250"]
```
[the remark this answers is not in the document]
Comment (darren221980) [reply environments/demo/demo.tfvars]: Now updated.

File: environments/demo/demo.tfvars [hunk 3]
```diff
@@ -1,7 +1,8 @@
 env                       = "demo"
 subscription              = "demo"
-app_gw_private_ip_address = ["10.51.32.121", "10.51.32.125"]
-cft_apps_cluster_ips      = ["10.51.31.250"]
+app_gw_private_ip_address = ["10.50.97.121", "10.50.97.125"]
+cft_apps_cluster_ips      = ["10.50.70.250", "10.50.95.250"]
```
Comment (thomast1906) [inline environments/demo/demo.tfvars:4]: `cft_apps_cluster_ips` [1] doesn't look correct

**Review by thomast1906 — APPROVED** [review]

File: environments/demo/demo.tfvars [hunk 4]
```diff
@@ -1,13 +1,17 @@
 env                       = "demo"
 subscription              = "demo"
-app_gw_private_ip_address = ["10.51.32.121", "10.51.32.125"]
-cft_apps_cluster_ips      = ["10.51.31.250"]
+app_gw_private_ip_address = ["10.50.97.121", "10.50.97.125"]
+cft_apps_cluster_ips      = ["10.50.79.250", "10.50.95.250"]
```
Comment (thomast1906) [inline environments/demo/demo.tfvars:4]: 10.50.49.246

Comment (darren221980) [reply environments/demo/demo.tfvars]: now fixed

File: environments/demo/demo.tfvars [hunk 5]
```diff
@@ -1,14 +1,16 @@
 env                       = "demo"
 subscription              = "demo"
-app_gw_private_ip_address = ["10.51.32.121", "10.51.32.125"]
-cft_apps_cluster_ips      = ["10.50.79.246"]
+app_gw_private_ip_address = ["10.50.97.121", "10.50.97.125"]
+cft_apps_cluster_ips      = ["10.50.79.246", "10.50.95.250"]
+cft_apps_ag_ip_address    = "10.50.97.123"
```
Comment (thomast1906) [inline environments/demo/demo.tfvars:5]: To be removed afaik

File: environments/demo/demo.tfvars [hunk 6]
```diff
@@ -1,14 +1,16 @@
...
 certificate_name_check    = false
 
 data_subscription       = "1c4f0704-a29e-403d-b719-b90c34ef14c9"
 privatedns_subscription = "1baf5470-1c3e-40d3-a6f7-74bfbce4b348"
 oms_env                 = "nonprod"
 
-hub = "nonprod"
```
Comment (thomast1906) [inline environments/demo/demo.tfvars:11]: value needed for apim

File: environments/demo/demo.tfvars [hunk 7]
```diff
@@ -1,14 +1,16 @@
...
+#FrontDoor access policy
+add_access_policy = false
```
Comment (thomast1906) [inline environments/demo/demo.tfvars:13]: remove

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**